### ISSUE TITLE
[Feature Request] Temporarily whitelist users for captcha solution

### ISSUE THREAD

**Issue by Folas1337** [BlackDemonZyT/BotSentry#116]
Currently if you are using a VPN and you connect, you are obviously blocked and then you get told to solve the captcha. This gets the VPN IP re-evaluated and most of the time just ends up in the IP getting blacklisted again and the user gets the captcha once more which feels to the end user like they're stuck in limbo. I'd like to handle this situation differently, at least with an option in the config.

I'd like to temporarily (for like 72 hours or also something configurable) whitelist the user who has solved the captcha (this obviously requires you to provide your username in the process). This way it can't be abused too much since it's time limited and you can still use a VPN if you want to, which I personally want to respect on my server because some people can't access the Internet freely or they want to protect their privacy.

In other words, give us a way to grant people access that still want to use a VPN without disabling the whole VPN protection feature of the plugin :)

For reference, this has also been discussed with @Laurenshup in ticket-1175 on Discord.

**Comment by Laurenshup** [comment]
The problem I am now seeing with this, is that normally people are only whitelisted after playing for X minutes. There is quite a difficult way of actually telling the way why they would whitelist themselves: are they using a VPN/were they blocked because of a different attack. Basically, I would have to disable the automatic whitelisting, which is quite hard to do.

**Comment by Folas1337** [comment]
> The problem I am now seeing with this, is that normally people are only whitelisted after playing for X minutes. There is quite a difficult way of actually telling the way why they would whitelist themselves: are they using a VPN/were they blocked because of a different attack. Basically, I would have to disable the automatic whitelisting, which is quite hard to do.

I hope I am not meant to give a response to that as I'm not sure how to give one... or if this is a final statement of sorts?
Is this just some "note to self" type of thing, I'm really confused 😅
